pkg/reconciler/timeout_handler.go

```diff
@@ -45,8 +45,8 @@ type StatusKey interface {
 	GetRunKey() string
 }
 
-// backoff contains state of exponential backoff for a given StatusKey
-type backoff struct {
```

Comment:
Was this exported because `GetBackoff` is exported below? Can we unexport `GetBackoff` instead?

Reply:
@ImJasonH I should have looked :angel: let's see :stuck_out_tongue:

Reply:
We are using `GetBackoff` in the reconcilier :thinking: